examples/hybrid/driver.jl

```diff
@@ -206,7 +192,11 @@ end
 dss_func = make_dss_func(comms_ctx)
 save_to_disk_func = make_save_to_disk_func(output_dir, is_distributed)
 
-dss_callback = FunctionCallingCallback(dss_func, func_start = true)
+dss_callback = FunctionCallingCallback(func_start = true) do Y, t, integrator
+    p = integrator.p
+    Spaces.weighted_dss!(Y.c, p.ghost_buffer.c)
+    Spaces.weighted_dss!(Y.f, p.ghost_buffer.f)
+end
```

Comment:
Why are we not using `dss_func` anymore (returned from `make_dss_func`)? Should we delete that if it's no longer used?